pkg/localstorev2/uploadstore.go

```diff
@@ -50,7 +50,7 @@ func (db *DB) Upload(ctx context.Context, pin bool, tagID uint64) (PutterSession
 		}),
 		done: func(address swarm.Address) error {
 			defer db.clearDirty(tagID)
-
+			defer db.events.Trigger(subscribePushEventKey)
```

Comment:
there is a nuance here and that this defer should be above the clearDirty()

Reply:
I think its better to clear first as if this trigger somehow starts an iteration before the clearDirty happens the iteration will stop due to the dirtyTag and no new trigger would come.